java/src/main/java/com/connexta/transformation/TransformApiDelete.java

```diff
@@ -0,0 +1,38 @@
+package com.connexta.transformation;
+
+import com.connexta.transformation.rest.models.ErrorResponse;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Validated
```

Comment:
✏️ Let's add a comment indicating how to re-generate this file and what is being added to it. (same for all other interfaces)